doc/releases/changelog-0.30.0.md

```diff
@@ -1,146 +1,285 @@
-  [(3976)](https://github.com/PennyLaneAI/pennylane/pull/3976)
+  op = qml.RX(np.pi / 2, wires=0)
 
-* Improved efficiency of `tapering()`, `tapering_hf()` and `clifford()`.
-  [(3942)](https://github.com/PennyLaneAI/pennylane/pull/3942)
+  with qml.tape.QuantumTape() as tape:
+      qml.ctrl(op, control=control_wires)
+  
+  with qml.tape.QuantumTape() as zyz_tape:
+      qml.RZ(np.pi / 2, wires=0)
+      qml.RY(np.pi / 4, wires=0)
+      qml.MultiControlledX(wires=control_wires + [0], control_values="11111", work_wires=[6, 7, 8])
+      qml.RY(-np.pi / 4, wires=0)
+      qml.MultiControlledX(wires=control_wires + [0], control_values="11111", work_wires=[6, 7, 8])
+      qml.RZ(-np.pi / 2, wires=0)
+  ```
+  
+  Fewer CNOT gates are used:
 
-* Improve the peak memory requirements of `tapering()` and `tapering_hf()` when used for larger observables.
-  [(3977)](https://github.com/PennyLaneAI/pennylane/pull/3977)
+  ```pycon
+  >>> tape.expand(depth=5).specs["gate_types"]["CNOT"]
+  60
+  >>> zyz_tape.expand(depth=5).specs["gate_types"]["CNOT"]
+  144
+  ```
+  
+  The decompositions are applied automatically when expanding tapes or decomposing operations in
+  PennyLane, but can also be accessed directly using
+  [ctrl_decomp_bisect()](https://docs.pennylane.ai/en/stable/code/api/pennylane.ops.op_math.ctrl_decomp_bisect.html).
 
-* Update Pauli arithmetic to more efficiently convert to a Hamiltonian.
-  [(#3939)](https://github.com/PennyLaneAI/pennylane/pull/3939)
+* A new decomposition to `qml.SingleExcitation` has been added that halves the number of
+  CNOTs required.
+  [(3976)](https://github.com/PennyLaneAI/pennylane/pull/3976)
+
+  ```pycon
+  >>> qml.SingleExcitation.compute_decomposition(1.23, wires=(0,1))
+  [Adjoint(T(wires=[0])),
+   Hadamard(wires=[0]),
+   S(wires=[0]),
+   Adjoint(T(wires=[1])),
+   Adjoint(S(wires=[1])),
+   Hadamard(wires=[1]),
+   CNOT(wires=[1, 0]),
+   RZ(-0.615, wires=[0]),
+   RY(0.615, wires=[1]),
+   CNOT(wires=[1, 0]),
+   Adjoint(S(wires=[0])),
+   Hadamard(wires=[0]),
+   T(wires=[0]),
+   Hadamard(wires=[1]),
+   S(wires=[1]),
+   T(wires=[1])]
+  ```
 
-* The adjoint differentiation method now supports more operations, and does no longer decompose
+* The adjoint differentiation method now supports more operations, and no longer decomposes
   some operations that may be differentiated directly. In addition, all new operations with a
-  generator are now supported by the method.
+  generator now support adjoint differentiation.
   [(#3874)](https://github.com/PennyLaneAI/pennylane/pull/3874)
 
-* When using `jax.jit` with gradient transforms, the trainable parameters are set correctly (instead of every parameter having
+* When using `jax.jit` with gradient transforms, the trainable parameters are now set correctly (instead of every parameter having
   to be set as trainable), and therefore the derivatives are computed more efficiently.
   [(#3697)](https://github.com/PennyLaneAI/pennylane/pull/3697)
```

Comment:
Minor, but I recommend leading this bullet points with the _user impact_ first, followed by the technical change (if needed). Less reading to identify what the improvement is.

Reply:
```suggestion
* Derivatives are computed more efficiently when using `jax.jit` with gradient transforms; the trainable parameters are now set  
correctly instead of every parameter having to be set as trainable.
  [(#3697)](https://github.com/PennyLaneAI/pennylane/pull/3697)
```

Reply:
Fixed: d11d104eb93d7d5c6c8fb1baae244ca474e94134